src/platform/intel/cavs/include/cavs/drivers/sideband-ipc.h

```diff
@@ -0,0 +1,44 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
+ *         Keyon Jie <yang.jie@linux.intel.com>
+ *         Rander Wang <rander.wang@intel.com>
+ */
+
+/**
+ * \file platform/intel/cavs/include/cavs/drivers/sideband-ipc.h
+ * \brief Defines layout of IPC registers for cAVS platforms with sideband IPC.
+ */
+
+#ifndef __INTEL_DRIVERS_SIDEBAND_IPC_H__
```

Comment:
Naming should be aligned with the rest of headers: `#ifndef __CAVS_DRIVERS_SIDEBAND_IPC_H__`.

Reply:
Fixed